engines/wintermute/detection.cpp

```diff
@@ -210,6 +214,1421 @@ class WintermuteMetaEngine : public AdvancedMetaEngine {
+			gameKeyMap->addAction(act);
+
+			act = new Action(kStandardActionOpenSettings, _("Music menu"));
+			act->setKeyEvent(KeyState(KEYCODE_m, 'm'));
+			act->addDefaultInputMapping("m"); // original keyboard
+			act->addDefaultInputMapping("JOY_RIGHT"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action("SCRLUP", _("Scroll up"));
+			act->setMouseWheelUpEvent();
+			act->addDefaultInputMapping("MOUSE_WHEEL_UP"); // original mouse
+			act->addDefaultInputMapping("UP"); // extra keyboard
+			act->addDefaultInputMapping("JOY_UP"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action("SCRLDN", _("Scroll down"));
+			act->setMouseWheelDownEvent();
+			act->addDefaultInputMapping("MOUSE_WHEEL_DOWN"); // original mouse
+			act->addDefaultInputMapping("DOWN"); // extra keyboard
+			act->addDefaultInputMapping("JOY_DOWN"); // extra joy
+			gameKeyMap->addAction(act);
+		} else if (gameId == "wtetris") {
+			act = new Action("ROTATE", _("Rotate"));
+			act->setKeyEvent(KEYCODE_UP);
+			act->addDefaultInputMapping("UP"); // original keyboard
+			act->addDefaultInputMapping("SPACE"); // original keyboard
+			act->addDefaultInputMapping("MOUSE_RIGHT"); // extra mouse
+			act->addDefaultInputMapping("JOY_UP"); // extra joy
+			act->addDefaultInputMapping("JOY_Y"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action(kStandardActionMoveLeft, _("Left"));
+			act->setKeyEvent(KEYCODE_LEFT);
+			act->addDefaultInputMapping("LEFT"); // original keyboard
+			act->addDefaultInputMapping("KP4"); // original keyboard
+			act->addDefaultInputMapping("MOUSE_WHEEL_UP"); // extra mouse
+			act->addDefaultInputMapping("JOY_LEFT"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action(kStandardActionMoveRight, _("Right"));
+			act->setKeyEvent(KEYCODE_RIGHT);
+			act->addDefaultInputMapping("RIGHT"); // original keyboard
+			act->addDefaultInputMapping("KP6"); // original keyboard
+			act->addDefaultInputMapping("MOUSE_WHEEL_DOWN"); // extra mouse
+			act->addDefaultInputMapping("JOY_RIGHT"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action("DROP", _("Drop"));
+			act->setKeyEvent(KEYCODE_DOWN);
+			act->addDefaultInputMapping("DOWN"); // original keyboard
+			act->addDefaultInputMapping("KP2"); // original keyboard
+			act->addDefaultInputMapping("MOUSE_MIDDLE"); // extra mouse
+			act->addDefaultInputMapping("JOY_DOWN"); // extra joy
+			gameKeyMap->addAction(act);
+		} else if (gameId == "zbang") {
+			act = new Action("ACTNXT", _("Next action"));
+			act->setRightClickEvent();
+			act->addDefaultInputMapping("MOUSE_RIGHT"); // original mouse
+			act->addDefaultInputMapping("MOUSE_WHEEL_UP"); // original mouse
+			act->addDefaultInputMapping("LEFTBRACKET"); // original keyboard
+			act->addDefaultInputMapping("JOY_B"); // extra joy
+			gameKeyMap->addAction(act);
+
+			act = new Action("ACTPRV", _("Previous action"));
+			act->setRightClickEvent();
```

Comment:
In addition to what I said about Pizza Morgana, both "ACTNXT" and "ACTPRV" are set to send right click events, which doesn't seem correct.

Reply:
Thanks. Fixed, see above.